### PR TITLE
add Euro class and .project file

### PR DESCRIPTION
**Pull request by quangkhai28**
I added euro classe and the files .project + .gitignore

### CODE REVIEW

File: src/test/EuroTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package test;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import com.Euro;
+
+public class EuroTest {
+
```
Comment (kakaict) [inline src/test/EuroTest.java]: add more tests

Comment (quangkhai28) [inline src/test/EuroTest.java]: no, all tests were added.

Comment (kakaict) [inline src/test/EuroTest.java]: i close pull request, but do not merge, start again

**Comment by quangkhai28** [comment]
fuck, why

**Comment by kakaict** [comment]
stop , redo again

**Pull request closed without merging**